Assets/Scripts/Controllers/Events/TradeController.cs

```diff
@@ -107,9 +107,9 @@ private void TrasfertTradedItems(Trade trade, Vector3 tradingCoordinates)
         {
             if (tradeItem.TradeAmount > 0)
             {
-                Tile tile = WorldController.Instance.World.GetFirstTileWithNoInventoryAround(6, (int)tradingCoordinates.x, (int)tradingCoordinates.y, (int)tradingCoordinates.z);
+                Tile tile = WorldController.Instance.World.GetTileAt((int)tradingCoordinates.x, (int)tradingCoordinates.y, (int)tradingCoordinates.z);
                 Inventory inv = new Inventory(tradeItem.Type, tradeItem.TradeAmount, tradeItem.TradeAmount);
-                WorldController.Instance.World.inventoryManager.PlaceInventory(tile, inv);
+                WorldController.Instance.World.inventoryManager.PlaceInventoryAround(tile, inv);
```

Comment:
Okay, that's fair, I've allowed for the default search radius to be 3, but it can be overridden, and I set the TradeController to use 6.